src/Models/RacingGame.js

```diff
@@ -0,0 +1,71 @@
+import { Car } from './';
+import { MESSAGE, RACING_GAME, UTIL } from '../constants';
+import {
+  getRacingResult,
+  getRandomIntInRange,
+  findMaxDistance,
+  findWinners,
+  parseGameResult,
+} from '../utils';
+
+export class RacingGame {
+  #cars;
+  #gameProgress;
+  #gameResult;
+
+  constructor(carNames, totalRounds) {
+    this.#cars = [];
+    this.#gameProgress = RACING_GAME.PROGRESS_TITLE;
+    this.#settingRacingGame(carNames, totalRounds);
+  }
+
+  #settingRacingGame(carNames, totalRounds) {
+    for (let carName of carNames) this.#cars.push(new Car(carName));
```

Comment:
고전적인 for문, 확장된 for문도 좋지만 좀 더 선언적으로 작성하면 좋겠네요! 왜냐하면, 이러한 코드는 제3자가 구문해석을 해야하기 때문에 글을 더 쉽게 읽도록 만들어주는 게 좋습니다.

```suggestion
    this.#cars = carNames.map((carName) => new Car(carName));
```